packages/core/src/Stepper.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Flex from './Flex'
+import Step from './Step'
+
+const StyledFlex = styled(Flex)`
+  & > :not(:last-child) {
+    margin-right: 32px;
+  }
+`
+
+function Stepper({ className, currentIndex, items, children }) {
+  return (
+    <StyledFlex className={className}>
+      {children}
+      {items &&
+        items.map((item, index) => (
+          <Step
+            key={index}
```

Comment:
array index should never be used as a key
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318 as linked from https://reactjs.org/docs/lists-and-keys.html

Reply:
I don't love using `index` but since the steps don't have an id and the name doesn't have to be unique (although they really should be) index all we really have and it seems fine to me in this scenario since the order of the items in a stepper will never change.

Quote from the react docs: "We don’t recommend using indexes for keys if the order of items may change."

However, all this being said if you can think of an alternative way then I will happily change it because while using index is okay in this scenario it might lead to others using it when it is not acceptable.

Reply:
if ever a step were to get introduce between existing steps, react would go haywire. perhaps requiring a step id is the sanest approach

Reply:
We could always remove this functionality and make it the responsibility of the consumer to map the children using `Stepper.Step`

Reply:
that seems sensible